ios/Classes/codec/AblyPlatformConstants.m

```diff
@@ -70,6 +70,14 @@
 NSString *const TxMessageData_data = @"data";
 NSString *const TxMessageData_type = @"type";
 
+// key constants for DeltaExtras
+NSString *const TxDeltaExtras_format = @"format";
```

Comment:
Uniqueness should not be a concern from what I can see!


In case you haven't looked at the usage in depth, here's how the story goes... :

`dart` platform message encoder encodes `DeltaExtras` as a json by putting value for `format` against `TxDeltaExtras.format` key. Now, `cocoa` or `java` extracts that value using `TxDeltaExtras_format` or `TxDeltaExtras.format` respectively. And vice versa is also true, i.e. when the encoding happens on platform side and decoded on dart side.

Reply:
In case you haven't noticed, these constants are generated.
You can read more about them here https://github.com/ably/ably-flutter/tree/main/bin#code-generation-to-keep-platform-constants-in-sync

Reply:
I haven't seen that before or noticed those were generated. So what you're saying is `format` and `from` are already scoped to `DeltaExtras`, there is no point prepending `"DeltaExtras"` to it. If there are no chances of collision, then I'm happy for you to resolve this conversation. Thanks

However, could we update the readme there to say that `codegencontext.dart` needs to be updated with new values, the `generating files` section is not just as simple as `cd bin` and `dart codegen.dart`.  found the readme a bit hard to understand, and had to use this PR to piece it together. Perhaps a `# Usage` section would be good for fresh eyes.

Reply:
We do have a section describing, briefly, the same: https://github.com/ably/ably-flutter/tree/main/bin#when-would-i-need-to-run-code-generation

We can re-organize if necessary. Or, would it be easier of there are more details with steps on how to create new constants? We can move this discussion to an issue, if that is the case